schemas/user.yml

```diff
@@ -65,6 +65,18 @@
       example: S-1-5-21-202424912787-2692429404-2351956786-1000
       description: >
         Unique identifier of the user.
+      otel:
+        - relation: match
+      otel_reuse:
```

Comment:
is this an equivalent of embed feature?

Reply:
yes, for some of the fields that are defined in ECS as reusable (like `process.user.name`) we defined explicit fields in SemConv. Since those do not exist explicitly in the ECS schema we need this additional concept of `otel_reuse` to document the mapping of those explicit attributes with implicit ECS fields